### ISSUE TITLE
Design / Styling Contributing?

### ISSUE THREAD

**Issue by EliotSlevin** [docker/kitematic#2747]
Hi,

I'm a designer who's been looking for some interfaces to sink his teeth into. This project fits all my requirements, electron based (so I can do the styling myself), is a useful tool for real world people, and has a complicated UI.

Aesthetically this app is really decent, but hasn't had a facelift for two years, and I think it could look even better.

I read the contributing guidelines, but there wasn't anything too specific for design contribution - is there anything I should know / people I should get in contact with?

**Comment by hellonearthis** [comment]
Have you worked with LESS that would help with knowing what can be done in the way of styling.
One feature needed is alternative colour themes*  (themes is also wanted).  
There is a lack of contrast in the current design that could be addressed by this.

Have you go an mockups on how your imagine your design changes?

**Comment by EliotSlevin** [comment]
Yes, I've worked with LESS before, and more extensively with pre-processors in general building a SASS ui library.

The main things I want to improve is typography (readability, and sizing), a few strange UX oddities (such as the [new tab / button](https://github.com/docker/kitematic/issues/1973)), and maybe an alternative dark theme. 

Will make a mockup.

**Comment by hellonearthis** [comment]
Would love to see that,  the 6 shades of gray for text is currently very low contrast and adding new colours would help imho.

Check out the [UX posts ](https://github.com/docker/kitematic/labels/ux) for issues people would like addressed.
